helpers/application_helpers.rb

```diff
@@ -1,6 +1,10 @@
 module ApplicationHelpers
   def github_file_url(file_path, version)
-    "https://github.com/thoughtbot/neat/blob/v#{version}/core/#{file_path}"
+    if version.to_s[0] == "1"
```

Comment:
Without knowing what `version` is here (a float?) you could write something like:

```
if version.to_s.start_with("1")
```

or...

```
if version.floor == 1
```